knowledge-base/kb-gridview-modify-foreground-selected-hovered-row.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Modify Foreground of Selected/Hovered GridViewRow
+description: Change Foreground of the cells when a row is selected or mouseovered
+type: how-to
+page_title: Update the Foreground of a row, which is Selected or Hovered
+slug: kb-gridview-modify-foreground-selected-hovered-row
+position: 0
+tags: modify, foreground, selected, hovered, row
+res_type: kb
+---
+
+## Environment
+<table>
+    <tbody>
+	    <tr>
+	    	<td>Product</td>
+	    	<td>RadGridView for WPF</td>
+	    </tr>
+    </tbody>
+</table>
+
+## Description
+
+How to change the Foreground of а GridViewRow that is selected or hovered. 
+
+## Solution
+
+1. Add a style targeting __GridViewRow__ with Triggers that check the __IsSelected__ and __IsMouseOver__ properties and change its __Foreground__. 
```

Comment:
We don't need a list if we have only one step. Can you remove this and write the step without the list item?

Also, can you also add a link to the Edit Control Templates article and add a note that this example is relevant to the Fluent theme and it may not be applicable to other themes? Consider including this information also in the title (the metadata) of the article.